tests/session/cmd/test_legacy.py

```diff
@@ -28,6 +28,32 @@ def test_legacy_list_default(tox_project: ToxProjectCreator, mocker: MockerFixtu
     assert outcome.state.options.show_core is False
 
 
+@pytest.mark.parametrize(
+    "configuration",
+    [
+        pytest.param("", id="missing toxenv section"),
+        pytest.param("[toxenv]", id="missing envlist"),
+        pytest.param("[toxenv]\nenv_list=", id="empty envlist"),
+    ],
+)
+def test_legacy_list_env_with_empty_or_missing_env_list(tox_project: ToxProjectCreator, configuration: str) -> None:
+    """we want to stay backwards compatible with tox 3 and show no output"""
+    outcome = tox_project({"tox.ini": configuration}).run("le", "-l")
+
+    outcome.assert_success()
+    outcome.assert_out_err("", "")
+
+
+def test_legacy_list_env_with_no_tox_file(tox_project: ToxProjectCreator) -> None:
+    project = tox_project({"pytest.ini": ""})
```

Comment:
You can leave the dictionary here empty.

Reply:
Ah, thanks! Thanks, I first got an "positional argument files ..." error, but I must have missed the empty dict before.